extensions/amp-story/0.1/media-pool.js

```diff
@@ -241,6 +301,55 @@ export class MediaPool {
     });
   }
 
+  /**
+   * @param {!HTMLMediaElement} mediaEl
+   * @private
+   */
+  bubbleEvents_(mediaEl) {
+    // TODO(alanorozco): Keep track of unlisteners for disposal.
+    const onNonBubblingEvent = this.onNonBubblingEvent_.bind(this, mediaEl);
+    NON_BUBBLING_EVENTS.forEach(type =>
+      listen(mediaEl, type, onNonBubblingEvent.bind(this, type)));
+  }
+
+  /**
+   * @param {!HTMLMediaElement} mediaEl
+   * @param {string} type
+   * @param {!Event} event
+   * @private
+   */
+  onNonBubblingEvent_(mediaEl, type, event) {
+    if (event.bubbles) {
+      // In case of platform differences.
+      return;
+    }
+    const {parentNode} = mediaEl;
+    if (!parentNode) {
+      // Detached, can't bubble.
```

Comment:
Will `mediaEl` always be detached by as a single element, or will it ever be detached inside a wrapper element? Checking `parentNode` won't work for the latter case.

Reply:
It's always detached as a single element.

Removed comment and updated some naming to be more clear about the intent here.